### PR TITLE
[Improvement-#9680][task-plugin] add hive sql log listener, print Map…

### LINKED ISSUE #9680: [Improvement][task-plugin] Add progress log during hive SQL execution
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

During the execution of hive SQL task, print MapReduce progress log and obtain application_id allows developers to better track tasks

### Use case

When the hive SQL task is executed, the MapReduce progress log can be obtained through the log viewing interface

### Related issues

no

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fengjian1129**
…Reduce progress log and obtain application_id allows developers to better track tasks

close #9680

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Comment by MonsterChenzhuo** [comment]
@fengjian1129 I feel that there is a part of your code that is not being submitted

File: dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/test/java/org/apache/dolphinscheduler/plugin/task/sql/HiveSqlLogThreadTest.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        System.setProperty("sun.security.krb5.debug", "true");
+        org.apache.hadoop.conf.Configuration configuration = new org.apache.hadoop.conf.Configuration();
+        configuration.set("hadoop.security.authentication", "Kerberos");
+        configuration.set("keytab.file", krb5KeyTabPath);
+        configuration.set("kerberos.principal", "vulcan@BJ.BAIDU.COM");
+
+        UserGroupInformation.setConfiguration(configuration);
+        UserGroupInformation.loginUserFromKeytab("vulcan", krb5KeyTabPath);
+
+        // 创建hive连接
```
Comment (EricGao888) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/test/java/org/apache/dolphinscheduler/plugin/task/sql/HiveSqlLogThreadTest.java]: Could u plz change this line of comment into English? The same for Line#91, thanks.

**Review by EricGao888 — CHANGES_REQUESTED** [review]

File: dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/test/java/org/apache/dolphinscheduler/plugin/task/sql/HiveSqlLogThreadTest.java (hunk 1 above)
Comment (fengjian1129) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/test/java/org/apache/dolphinscheduler/plugin/task/sql/HiveSqlLogThreadTest.java]: ok ， i will Resubmit

**Comment by fengjian1129** [comment]
@MonsterChenzhuo @zhuangchong @EricGao888 I have resubmitted the code。https://github.com/apache/dolphinscheduler/pull/11562

**Pull request closed without merging**